amplify.yml

```diff
@@ -7,12 +7,16 @@ applications:
             - nvm install 14
             - nvm use 14
             - node -v
+            - export FLUTTER_HOME=${HOME}/sdks/flutter
```

Comment:
I validated the changes to the build process by creating an amplify hosting project on my account. That can be seen here: https://docs-flutter-authenticator-component.d3j7wwah334ae5.amplifyapp.com/components/authenticator?platform=flutter

Reply:
Note: In the preview above, there are multiple Flutter Authenticators on the same page. As a result, the static assets required to render the Flutter Authenticator are being fetched multiple times. Aside from canvaskit.js & canvaskit.wasm (which flutter pulls in from unpkg.com by default, and caches for a year) the static resources total ~5kb. There are probably some improvements that can be made to prevent these from being fetched multiple times, but I'm going to consider that out of scope for now.